### PR TITLE
i18n(TDI-41939) : export (upload) i18n package to xtm

### PR DESCRIPTION
**Pull request by akhabali**

### CODE REVIEW

**Comment by gonchar-ivan** [comment]
just interesting
What is purpose of this image? 
'khabali/jenkins-java-build-container:latest'

**Comment by akhabali** [comment]
This image provides the build environment and contains mvn, java, docker and other required tools for the build like rsync.

The old image was a 32bit image and nodejs doesn't exist in 32 bit for Linux.

Also I wasn't able to found any existing image that already has all those tools

File: Jenkinsfile [hunk 1]
```diff
@@ -145,6 +157,73 @@ spec:
...
+        stage('Push to Xtm') {
+            when {
+                anyOf{
+                    expression { params.PUSH_TO_XTM == true }
+//                    allOf{
+//                        triggeredBy 'TimerTrigger'
+//                        expression {
+//                            (calendar.get(Calendar.WEEK_OF_MONTH) == 2 ||  calendar.get(Calendar.WEEK_OF_MONTH) == 4) && calendar.get(Calendar.DAY_OF_WEEK) == Calendar.THURSDAY
+//                        }
+//                    }
```
Comment (gonchar-ivan) [inline Jenkinsfile]: Don't forget to remove commented code someday ;-)

**Comment by gonchar-ivan** [comment]
Thanks for explanation about image.
Also could you, please, tell a bit about json files: 
talend-i18n.json
package.json

File: Jenkinsfile (hunk 1 above)
Comment (akhabali) [reply Jenkinsfile]: this part automate the export. it will be activated soon ;)

**Comment by akhabali** [comment]
It's all explained in the https://github.com/Talend/i18n-product project.
There is a node script that manage the export / import of i18n resources. we only do an integration of that in this PR

**Pull request merged**